CHANGELOG.md

```diff
@@ -1,17 +1,75 @@
 # Changelog
 
-## 23.1.0-RC2
+## 23.1.1
+This update is required for the Goerli Shanghai/Capella upgrade and recommended for all Mainnet users. If you use Besu on Goerli, update to 23.1.1. If you previously used 23.1.1-RC1, update to test 23.1.1 on Goerli. 
+
+### Breaking Changes
```

Comment:
nit: remove?

Reply:
This is here for parity with what will be released from the release-23.1.x branch.  I don't think this text is likely to change

Reply:
I just meant the empty Breaking Changes header but nbd